packages/calling/src/Voicemail/UcmBackendConnector.test.ts

```diff
@@ -64,6 +88,31 @@ describe('Voicemail UCM Backend Connector Test case', () => {
     });
   });
 
+  it('verify successful voicemailContent', async () => {
+    const voiceMailPayload = <VoicemailResponseEvent>(<unknown>mockVoicemailContentResponse);
+
+    webex.request.mockResolvedValueOnce(voiceMailPayload);
+    const response = await (await ucmBackendConnector.getVoicemailContent(messageId)).data;
```

Comment:
Do we require two await here?

Reply:
Not required, updated the code. Thanks for noticing!